src/Smart.Design.Razor/TagHelpers/SideMenu/SideMenuTagHelper.cs

```diff
@@ -39,11 +39,8 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
         {
             throw new ArgumentException($"{nameof(LabelAndLinks)} cannot be null or empty");
         }
-        var menu = new TagBuilder("div");
-        menu.AddCssClass("c-app-layout-inner__sidebar u-position-fixed");
-
-        menu.InnerHtml.SetHtmlContent(_sideMenuHtmlGenerator.GenerateListOfItems(LabelAndLinks));
 
+        var menu = _sideMenuHtmlGenerator.GenerateListOfItems(LabelAndLinks);
```

Comment:
Seeing the PR's description, I assume the div removed here will be directly added in code rather than in the tagHelper?

Reply:
Yes